test/http_binding_1.js

```diff
@@ -117,6 +132,21 @@ describe("HTTP Transport Binding - Version 1.0", () => {
   });
 
   describe("Binary", () => {
+    it('works with mTLS authentication', () => {
+      const event = new BinaryHTTPEmitter({
+        method: 'POST',
+        url: `${webhook}/json`,
+        httpsAgent: new https.Agent({
+          cert: 'some value',
+          key: 'other value'
+        })
+      })
+      return event.emit(cloudevent).then(response => {
```

Comment:
This can also be converted to ES6.

Reply:
Again - this was prior to dropping Node 6 and 8 support.